### DISCUSSION TITLE
Steam achievements option not working on Steam Deck

### DISCUSSION THREAD

**Discussion by Obseerve** [julianxhokaxhiu/FFNx#656] [Q&A] [answered]
Hey,

I tried enabling the Steam achievements option on the SYW launcher on Steam Deck but it keeps giving me that error "Steam Error - Steam must be running to play this game with achievements (SteamAPI_Init() failed).
I've tried within Steam but also in desktop mode with Steam launched but both failed.

If anyone have any clue how to fix this i'd be glad !

**Comment by julianxhokaxhiu** [comment, marked as answer]
Steam Achievements work fine when using https://github.com/dotaxis/7thDeck which installs 7th and uses FFNx. If you have issues with SYW please approach Satsuki on his support channels. Thank you!
